baking/setup.cfg

```diff
@@ -0,0 +1,30 @@
+# SPDX-FileCopyrightText: 2023 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
+
+[metadata]
+name = tezos-baking
+author = Serokell
+author_email = tezos-packaging@serokell.io
+description = Package that provides systemd services that orchestrate other services from Tezos packages
+license = LicenseRef-MIT-OA
```

Comment:
It works, but only since `61.0` (see [here](https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html)).
While `focal` has [package](https://packages.ubuntu.com/focal/python3-setuptools) with much lower version.
Current setup builds fine on both `ubuntu` and `fedora`, as well as local install.